Library/Basic Set/Basic Set Traits.adq

```diff
@@ -18063,6 +18103,107 @@
 				"points": 8
 			}
 		},
+		{
+			"id": "dd8c45d8-9789-461a-85d3-14762795b2a0",
+			"type": "trait",
+			"name": "Modular Abilities (Book-Learned Wisdom)",
+			"reference": "DF8:17,B71,P62",
+			"notes": "Prerequisite: Full literacy",
```

Comment:
Would consider changing this to a functional prerequisite for a trait named "Language", though I understand that iirc this does not take the necessary modifiers to be properly viable.

Reply:
All fixed, thanks for the comments. I tested the "Full Literacy" requisite and indeed it can be implemented.